lte/gateway/python/integ_tests/s1aptests/test_attach_auth_failure.py

```diff
@@ -0,0 +1,63 @@
+"""
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+
+import unittest
+
+import s1ap_types
+
+from integ_tests.s1aptests import s1ap_wrapper
+
+class TestAuthFailure(unittest.TestCase):
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_auth_failure_proc(self):
+        """ Testing of sending authentication failure procedure """
+        num_ues = 1
+
+        self._s1ap_wrapper.configUEDevice_ues_same_imsi(num_ues)
```

Comment:
Do we need configUEDevice_ues_same_imsi here or just configUEDevice is sufficient, given that we have only one UE